prow/jobs/incubator/hack-showcase/hack-showcase.yaml

```diff
@@ -1,8 +1,7 @@
 job_template: &job_template
   skip_report: false
-  run_if_changed: "^hack-showcase/"
+  always_run: true
```

Comment:
You should not use always_run: True for jobs which are not required for all PRs. Please correct and use run_if_changed.